shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/Serialization.hs

```diff
@@ -19,22 +25,35 @@ module Shelley.Spec.Ledger.Serialization
   , rationalFromCBOR
   , mapToCBOR
   , mapFromCBOR
+  -- Annotated Decoding
+  , AnnotatedDecoder (..)
+  , withAnnotationSlice'
+  , FromCBORAnnotated (..)
+  , decodeAnnotated
+  , withAnnotationSlice
+  , fromCBOREmptyAnnotation
+  , decodeWrapped
   )
 where
 
+import           Cardano.Prelude
+import           Prelude ()
```

Comment:
Do you genuinely want instances from `Prelude`? Otherwise, should use `NoImplicitPrelude` instead.

Also, we haven't used `cardano-prelude` anywhere else in this package, so I'm a little uncertain about its use here. Mixing preludes is likely to result in confusion. If there are things we need from it, I'd rather just import them explicitly.